### DISCUSSION TITLE
Contributions are gone after changing email address

### DISCUSSION THREAD

**Discussion by NoNews** [community/community#29339] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi.

I've changed my email address, and after it I realised that It might be a reason why all of my contributions are gone.

I've reverted these changes and had added removed email address back, however contributions are still not showed.

However, they are, if I call an API:
https://api.github.com/repos/android-ui-testing/Cookbook/contributors?page=1

But there is no contributions on the ui:
https://github.com/android-ui-testing/Cookbook

Is there a way to resolve it, or I just need to wait a bit after adding previous email address?

UPD: It didn't work, I did everything described on the documentation.

**Comment by samuel-ma** [comment]
Hi there, I've gone through the same thing 
The solution is actually pretty simple
Read through this documentation [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)

**Reply by NoNews** [reply]
Thanks! But I can see the contributions from my profile.

I can't see them for the particular repositories :(
I've already checked this guide and added an email I used for the commits back to the github, but it still doesn't work

**Reply by samuel-ma** [reply]
Have you verified the email address you reverted back 
Do that and check your commits for the particular repositories again and it should be a quick fix

**Reply by NoNews** [reply]
Yep, I've already done it. Still doesn't work :(

**Reply by samuel-ma** [reply]
Seriously....
lol, sorry bruh
I don't know what else to say
If all fails, please visit here as a last resort [click here](https://support.github.com/)
Hope you find the solution, I rest my case here.....

**Reply by NoNews** [reply]
Didn't help unfortunately. I did everything as described on the documentation.

Anyway, thank you!

**Comment by airtower-luna** [comment, marked as answer]
It looks like your commits are recognized as yours and you're listed as a contributor in the sidebar, just the "Insights" contributor list doesn't. After over a week I think it's reasonable to [send a bug report to GitHub support](https://support.github.com/contact/bug-report). Good luck!
